src/css/components/Button.scss

```diff
@@ -3,9 +3,9 @@
 // * @define default Button
 
 // Global default button colors that are themed
-$button-color:                    $cu-foreground !default;
-$button-border-color:                $cu-divider !default;
-$button-text-color:                  $cu-info !default;
+$button-color:                       white !default;
```

Comment:
If we want theming to work on v4 we'll want to abstract-ify colors similar to how we've done it currently. Themes will only apply to the team web app, but may be nice to consider earlier rather than later.